src/frontends/pytorch/src/op/arange.cpp

```diff
@@ -108,6 +109,9 @@ OutputVector translate_arange_fx(const NodeContext& context) {
     if (context.has_attribute("dtype")) {
         dtype = context.get_attribute<element::Type>("dtype");
     }
+    if (end.get_partial_shape().rank().is_dynamic()) {
```

Comment:
Why only in dynamic case? What if it is static and 1D?

Reply:
Makes sense to add this change but we haven't observed such an input on the models we are working on. I updated this line to include static 1D inputs for end.